inginious/frontend/pages/course_admin/student_list.py

```diff
@@ -24,19 +28,19 @@ def GET_AUTH(self, courseid):  # pylint: disable=arguments-differ
 
         if "download_audiences" in flask.request.args:
             audiences = [{"description": audience["description"],
-                           "students": audience["students"],
-                           "tutors": audience["tutors"]} for audience in
-                          self.user_manager.get_course_audiences(course)]
+                          "students": audience["students"],
+                          "tutors": audience["tutors"]} for audience in
+                         self.user_manager.get_course_audiences(course)]
             response = Response(response=yaml.dump(audiences), content_type='text/x-yaml')
             response.headers['Content-Disposition'] = 'attachment; filename="audiences.yaml"'
```

Comment:
There is actually no much interest in downloading the audiences in the YAML format while the upload format is CSV. Using the same file format could allow the user to simply add some people to the list without regenerating it manually. Either the ``id_field`` can be set to the email, either the user can be asked which field they prefer.